### PR TITLE
ENG-13845: Update README and add release workflow.

### PR DESCRIPTION
**Pull request by ccampo133**
## Description of the change

As title.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue).
- [x] New feature (non-breaking change that adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

### Development

- [x] Lint rules pass locally.
- [ ] The code changed/added as part of this pull request has been covered with tests.
- [ ] All tests related to the changed code pass.

### Code review

- [x] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.
- [x] Jira issue referenced in commit message and/or PR title.

### Testing

Manual testing TODO.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,21 +4,153 @@
...
+### Installation
+
+The Dmap CLI can be installed as a native binary, a Docker image, or from 
+source. Each approach is described below.
+
+#### Release Binaries
+
+Binary executables of the CLI are available for Linux, MacOS, and Windows
+platforms. The appropriate binary for your platform can be downloaded from the 
+[releases page](TODO), e.g.:
```
Comment (ccampo133) [inline README.md:89]: Reminder: todo

File: README.md [hunk 2]
```diff
@@ -4,21 +4,153 @@
...
+
+```bash
+curl -OL TODO
```
Comment (ccampo133) [inline README.md:92]: Reminder: todo

File: README.md [hunk 3]
```diff
@@ -4,21 +4,156 @@
...
+environment.
+
+Scanning a cloud environment with Dmap will provide you with a list of data
+repositories that exist in that environment. Scanning a data repository with
+Dmap will provide you with all the fields in the data repository that contain
+sensitive data.
+
+We use Open Policy Agent's (OPA) [Rego API](https://pkg.go.dev/github.com/open-policy-agent/opa/rego) 
+to define policies (termed "data labels") that can be used to classify sensitive
+data in data repositories. Dmap provides a set of [predefined data labels](classification/labels)
```
Comment (yoursnerdly) [inline README.md:34]:
```suggestion
to define rules that can be used to classify sensitive data in data repositories and assign them appropriate labels (such as CCN, SSN etc). Dmap provides a set of [predefined data labels](classification/labels) 
```

File: README.md [hunk 4]
```diff
@@ -4,21 +4,156 @@
...
+sha256sum -c dmap_<version>_sha256sums.txt
+gpg --verify dmap_<version>_sha256sums.txt.sig dmap_<version>_sha256sums.txt
+```
+
+#### Docker
+
+You can run the image directly as a container, e.g.:
+
+```bash
+docker run --rm public.ecr.aws/cyral/dmap:latest
```
Comment (yoursnerdly) [inline README.md:118]: Can we show a couple of dmap options (as example) in the docker run command?

**Review by yoursnerdly — APPROVED** [review]
Minor suggestions on the documentation. I presume the github actions are tested?

**Pull request merged**